text/0000-repository-level-ownership

```diff
@@ -0,0 +1,128 @@
+
+The section should return to the examples given in the previous section, and explain more fully how the detailed proposal makes those examples work.
+-->
+Instead of a list of repositories and ownership listed in a central place (currently [GOVERNANCE.md][gov]) each repository within the project will contain a `CODEOWNERS` file that lists owners for that particular repository.
+
+It is recommended that each repository `CODEOWNERS` file have at least one GitHub team owner as a "fallback" in case any individuals are unreachable.
+
+# Migration
+[migration]: #migration
+
+<!--
+This section should document breaks to public API and breaks in compatibility due to this RFC's proposed changes. In addition, it should document the proposed steps that one would need to take to work through these changes. Care should be give to include all applicable personas, such as platform developers, buildpack developers, buildpack users and consumers of buildpack images.
+-->
+- Remove list of repositories from [`GOVERNANCE.md`][gov].
+- Maintainers of current repositories should identify owners and create an `OWNERS` file in each repository.
+
+[gov]: https://github.com/buildpacks/community/blob/main/GOVERNANCE.md
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+Why should we *not* do this?
+
+# Alternatives
+[alternatives]: #alternatives
+<!--
+- What other designs have been considered?
+- Why is this proposal the best?
+- What is the impact of not doing this?
+-->
+- Continue to expect ownership of repositories to be only at the sub-team level.
+- Use _custom_ `OWNERS` and `OWNER_ALIAS` files instead of relying on GitHub specific functionality/standards.
+    - K8s does this:
+      > "We use aliases for groups instead of GitHub Teams, because changes to GitHub Teams are not publicly auditable."
+
+# Prior Art
+[prior-art]: #prior-art
+
+<!--
+Discuss prior art, both the good and bad.
+-->
+- [K8s' OWNERS](https://www.kubernetes.dev/docs/guide/owners/)
+- [Chromium's OWNERS](https://chromium.googlesource.com/chromium/src/+/HEAD/docs/code_reviews.md#owners-files)
+- [GitLab's CODEOWNERS](https://docs.gitlab.com/ee/user/project/code_owners.html#set-up-code-owners)
+
+
+# Unresolved Questions
+[unresolved-questions]: #unresolved-questions
+
+<!--
+- What parts of the design do you expect to be resolved before this gets merged?
+- What parts of the design do you expect to be resolved through implementation of the feature?
+- What related issues do you consider out of scope for this RFC that could be addressed in the future independently of the solution that comes out of this RFC?
+-->
+- Do owners have to be sub-team maintainers?
+    - This question is out-of-scope of this RFC and would require a larger more impacting governace change and discussion.
+- How are permissions handled?
+    - At this point in time, it's proposed that maintainers and leads take the appropriete actions necessary to ensure parity. This can easily be automated once adopted.
```

Comment:
```suggestion
    - At this point in time, it's proposed that maintainers and leads take the appropriate actions necessary to ensure parity. This can easily be automated once adopted.
```